src/include/stir/ProjDataInfo.h

```diff
@@ -349,6 +349,20 @@ class ProjDataInfo
   //! Return a string describing the object
   virtual std::string parameter_info() const;
   
+  //! Set horizontal bed position
+  void set_bed_position_horizontal(float &bed_position_horizontal_arg)
```

Comment:
never a reference as argument, unless you want to change the argument, or it's a `const&` for a large object that you don't want to copy